### PR TITLE
F#207 create package sr robot commander

### PR DESCRIPTION
**Pull request by AndriyPt**
Basic package files

### CODE REVIEW

File: sr_robot_commander/package.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  <url type="website">http://www.shadowrobot.com</url>
+  <url type="repository">https://github.com/shadow-robot/sr-ros-interface</url>
+  <url type="bugtracker">https://github.com/shadow-robot/sr-ros-interface/issues</url>
+
+  <author>Andriy Petlovanyy</author>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>roscpp</build_depend>
```
Comment (ugocupcic) [inline sr_robot_commander/package.xml]: will be python based

**Comment by ugocupcic** [comment]
Also add basic README

**Comment by ugocupcic** [comment]
happy to merge right after

**Comment by toliver** [comment]
Please do not merge yet

I'd like to launch the release before this changes get merged.

**Pull request merged**